rollup.config.js

```diff
@@ -0,0 +1,23 @@
+import babel from 'rollup-plugin-babel';
+import { sizeSnapshot } from 'rollup-plugin-size-snapshot';
+import pkg from './package.json';
+
+const input = './src/index.js';
+
+const external = id => !id.startsWith('\0') && !id.startsWith('/') && !id.startsWith('.');
+
+export default [
+  {
+    input,
+    output: { file: pkg.main, format: 'cjs', exports: 'named' },
+    external,
+    plugins: [babel({ plugins: ['external-helpers'] }), sizeSnapshot()],
+  },
+
+  {
+    input,
+    output: { file: pkg.module, format: 'es', exports: 'named' },
```

Comment:
Only difference between the two config is output.file and output.format — what do you think about DRYing this up a bit?